generators/app/templates/app/controllers/apiInfo.js

```diff
@@ -0,0 +1,22 @@
+const { merge } = require('lodash');
+const pjson = require('../../package.json');
+
+exports.apiInfo = (req, res, next) => {
+  try {
+    return res.status(200).send(res.headers);
+  } catch (err) {
+    return next(err);
+  }
+};
+
+exports.dependenciesInfo = (req, res) => {
+  try {
+    if (!req.body) {
+      req.body = {};
+    }
+    merge(req.body, { dependencies: pjson.dependencies });
```

Comment:
We need merge with body sent by client ? Also i don't know why is useful expose dependencies in an endpoint